src/routes/v1/articles.routes.js

```diff
@@ -0,0 +1,21 @@
+import dotenv from 'dotenv';
+import { Router } from 'express';
+import RatingsController from '../../controllers/ratings.controller';
+import authMiddleware from '../../middleware/auth.middleware';
+import validateRatings from '../../middleware/validation.middleware';
+
+// middleware
+const validateRequest = validateRatings();
+
+dotenv.config();
+
+const route = Router();
+
+route.patch(
```

Comment:
Good job @codexempire  but I do not see any validation to ensure the article author cannot rate his/her own articles?